ui/app/components/clients/dashboard.js

```diff
@@ -175,8 +217,12 @@ export default class Dashboard extends Component {
   }
 
   @action
-  resetData() {
-    this.barChartSelection = false;
-    this.selectedNamespace = null;
+  selectStartMonth(month) {
```

Comment:
these are fired off from the modal that sets the startMonth. Right now, they do nothing but set a param, when it's wired up they'll fire off a network request.